cmd/compose/pkg/usersapi/users.go

```diff
@@ -21,28 +21,22 @@ type Client struct {
 }
 
 func (cl *Client) GetUser(ctx context.Context, id string) (User, error) {
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet,
-		cl.BaseURL+fmt.Sprintf("/users/%s", id), nil)
+	var conn *grpc.ClientConn
+	conn, err := grpc.Dial(cl.BaseURL, grpc.WithInsecure())
```

Comment:
in GRPC the idea is the same as for HTTP. You don't need to dial each time, instead, you create a connection once and then reuse it. 
probably, I would inject "users grpc client" as dependency in the main.

Reply:
:+1: